src/interlocked_hl.c

```diff
@@ -2,7 +2,9 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 #include <inttypes.h>
-#include <limits.h>
+#include <stddef.h>                 // for NULL
+
+#include "umock_c/umock_c_prod.h"   // for MU_COUNT_ARG_0, MU_DISPATCH_EMPTY...
```

Comment:
Is the MU_* stuff actually why we include this? Sounds like macro_utils.h

Reply:
It's not. It's for the IMPLEMENT_MOCKABLE. Fixed by removing the "for"